src/Compilers/CSharp/Portable/Lowering/MethodToClassRewriter.cs

```diff
@@ -721,6 +721,17 @@ internal override void AddSynthesizedAttributes(ModuleCompilationState compilati
 
                 AddSynthesizedAttribute(ref attributes, this.DeclaringCompilation.TrySynthesizeAttribute(WellKnownMember.System_Diagnostics_DebuggerHiddenAttribute__ctor));
             }
+
+            internal override TypeSymbol IteratorElementType
+            {
+                get
+                {
+                    // BaseMethodWrapperSymbol should not be rewritten by the IteratorRewriter
+                    // https://github.com/dotnet/roslyn/issues/11649
```

Comment:
Checked my assumption and yes our code is is indeed littered with external links in the production code.  Clearly many developers believe this has value.  I see no reason to push back on this practice.